_topic_map.yml

```diff
@@ -262,6 +262,15 @@ Topics:
     File: configuring-google-identity-provider
   - Name: Configuring an OpenID Connect identity provider
     File: configuring-oidc-identity-provider
+- Name: Configuring certificates
+  Dir: certificates
+  Topics:
+  - Name: Replacing the default router certificate
```

Comment:
Aren't we calling router 'ingress' now? @bmcelvee can you confirm?

Reply:
You're correct. I spoke with her about this Friday afternoon. I'll update these references.

Reply:
These instances have been renamed to `ingress`.